exporter/exporter.go

```diff
@@ -503,7 +522,7 @@ func Clear(force bool) {
 		shouldBeRemoved := true
 		if !force {
 			for _, pmd := range exporter.parsedMetricsData {
-				if len(pmd.sequence) != 0 {
+				if len(pmd.sequences[0]) != 0 {
```

Comment:
How `sequences` is used differs between histogram and others. It makes the spec hard to understand.